test/core/image_processing/CMakeLists.txt

```diff
@@ -23,12 +23,13 @@ foreach(_name
 
     unset(_name)
     unset(_target)
+endforeach()
 
 foreach(_name
   lanczos_scaling
   )
-  set(_test t_image_processing_${_name})
-  set(_target test_image_processing_${_name})
+  set(_test t_${_name})
```

Comment:
FYI, I 'invented' and followed the convention of prefixing
- core tests with `t_core_` 
- extension tests with `t_ext_<extension name>_`
I personally like this distinction of core vs extension stuff.

However, don't bother about such trivial issues. I/We will run a unifying clean up at one point later. Just keep your test names unique.

Reply:
Should I resolve this then?

Reply:
Yes, certainly. This I added as single comment, just FYI, not review, no request for changes.